### ISSUE TITLE
A solution of "Received no valid replies at all"

### ISSUE THREAD

**Issue by tastypear** [h-a-s-k/clewd#2]
After the upgrade, I encountered the error message 'Received no valid replies at all' and there was no useful log output in the terminal.

I checked the discussions on Gitgud and rolled back to version 3.8.5.
The program then threw an error stating that the service was not available in my region.

At this point, I realized that the issue might be with the system proxy.
Upon inspecting the proxy settings, I found that my Proxifier had previously only been set to proxy 'node.exe', whereas the new program required including 'clewd-superfetch-win-amd64.exe'.

Of course, the ideal solution would be to directly proxy Claude's domain.
After configuring this, I ran version 4.4 again and everything worked as expected.

In summary, I was dealing with a network issue, and I used the error messages from the older version to pinpoint the problem.

Since I couldn't log in to Gitgud (the website doesn't allow me to log in via a proxy), I'm documenting the troubleshooting process here in hopes that it might help other users.

**Comment by PhiFever** [comment]
Another possible reason: Kaspersky's Internet Defense intercepts the app's web requests.You should close Kaspersky when using.
